docs/user-guide/api-mediation/configuration-extender-passtickets.md

```diff
@@ -134,11 +129,14 @@ Follow these steps to enable PassTicket Support specific to your ESM. Consult wi
         F ACF2,REBUILD(PTK),CLASS(P)
```

Comment:
Actually, I don't see the mention of MULTI-USE in the codeblock. Shouldn't it be in here?